src/sgame/sg_main.cpp

```diff
@@ -2180,18 +2180,28 @@ void G_CheckVote( team_t team )
 
 	if ( !quorum )
 	{
-		cmd = va( "print_tr %s %d %d", ( team == TEAM_NONE ) ? QQ( N_("Vote failed ($1$ of $2$; quorum not reached)") ) : QQ( N_("Team vote failed ($1$ of $2$; quorum not reached)") ),
-		            level.team[ team ].voteYes + level.team[ team ].voteNo, level.team[ team ].numPlayers );
+		cmd = va( "print_tr %s %d %d", ( team == TEAM_NONE )
+			? QQ( N_("Vote failed") " " N_("($1$ of $2$; quorum not reached)") )
+			: QQ( N_("Team vote failed") " " N_("($1$ of $2$; quorum not reached)") ),
+			level.team[ team ].voteYes + level.team[ team ].voteNo,
+			level.team[ team ].numPlayers );
 	}
 	else if ( pass )
 	{
-		cmd = va( "print_tr %s %d %d", ( team == TEAM_NONE ) ? QQ( N_("Vote passed ($1$ — $2$)") ) : QQ( N_("Team vote passed ($1$ — $2$)") ),
-		            level.team[ team ].voteYes, level.team[ team ].voteNo );
+		cmd = va( "print_tr %s %d %d", ( team == TEAM_NONE )
+			? QQ( N_("Vote passed") " " N_("($1$ — $2$)") )
```

Comment:
It may be hard to tell what `($1$ — $2$)` represents in isolation

Reply:
We don't need to know what it represents, only that this typography is used. It's the same issue like translating english `%s: %s` to french `%s : %s`, the context and the words should have no importance. It's about translating typography, not translating words.

There may be problems right-to-left language (which one is read first, right? or left?), but that problem is not added by this change and already exists in the original string.